notebooks/multilingual_rag_podcast.ipynb

```diff
@@ -0,0 +1,10205 @@
+{
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "xRFiZ-FMQ68a"
+      },
+      "source": [
+        "# 🇮🇹🇬🇧 Multilingual RAG from a 🎧 podcast\n",
+        "\n",
+        "<a target=\"_blank\" href=\"https://colab.research.google.com/github/deepset-ai/haystack-cookbook/blob/main/notebooks/multilingual_rag_from_podcast.ipynb\">\n",
+        "  <img src=\"https://colab.research.google.com/assets/colab-badge.svg\" width=\"200\" alt=\"Open In Colab\"/>\n",
+        "</a>\n",
+        "\n",
+        "*Notebook by [Stefano Fiorucci](https://github.com/anakin87)*\n",
+        "\n",
+        "This notebook shows how to create a multilingual Retrieval Augmented Generation application application, starting from a podcast.\n",
+        "\n",
+        "🧰 **Stack**:\n",
+        "- Haystack LLM framework\n",
+        "- OpenAI Whisper model for audio transcription\n",
+        "- Qdrant vector database\n",
+        "- multilingual embedding model: multilingual-e5-large\n",
+        "- multilingual LLM: Mixtral-8x7B-Instruct-v0.1"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "toc",
+        "id": "ugelCfVFYpOd"
+      },
+      "source": [
+        ">[🇮🇹🇬🇧 Multilingual RAG from a 🎧 podcast](#scrollTo=xRFiZ-FMQ68a)\n",
```

Comment:
I don't think this table of contents will work. Let's try it, but we may have to remove